shared/chat/conversation/info-panel/header.js

```diff
@@ -1,23 +1,25 @@
 // @flow
 import * as React from 'react'
-import {Avatar, Box, ClickableBox, Text} from '../../../common-adapters'
+import {Avatar, Box, ClickableBox, Icon, Text} from '../../../common-adapters'
 import {globalMargins, globalStyles, isMobile} from '../../../styles'
 
 type SmallProps = {
   teamname: string,
   participantCount: number,
   onClick: () => void,
+  onClickGear: any => void,
 }
 
-const SmallTeamHeader = ({teamname, participantCount, onClick}: SmallProps) => (
+const gearIconSize = isMobile ? 24 : 16
+
+const SmallTeamHeader = ({teamname, participantCount, onClick, onClickGear}: SmallProps) => (
   <ClickableBox
     style={{
       ...globalStyles.flexBoxRow,
       alignItems: 'center',
       marginLeft: globalMargins.small,
-      marginTop: globalMargins.small,
     }}
-    onClick={onClick}
+    onClick={evt => !evt.defaultPrevented && onClick()}
```

Comment:
Perhaps condition on `isMobile` here for consistency with the below? Or maybe before returning the elements,
```
if (!isMobile) {
  onClick = evt => ...
  onClickGear = evt => ...
}
```

Reply:
The one below should actually be calling `preventDefault` regardless of `isMobile`, fixed that and factored them out!